package.json

```diff
@@ -20,27 +21,25 @@
     "node": "18.16.0"
   },
   "devDependencies": {
-    "@babel/core": "7.20.12",
-    "@babel/eslint-parser": "7.19.1",
     "@intlify/eslint-plugin-vue-i18n": "1.4.0",
-    "@types/eslint": "^8.40.0",
+    "@openverse/eslint-plugin": "workspace:*",
+    "@types/jest": "26.0.22",
     "@types/node": "17.0.32",
-    "@typescript-eslint/eslint-plugin": "5.54.0",
-    "@typescript-eslint/parser": "5.54.0",
+    "@typescript-eslint/eslint-plugin": "5.60.0",
+    "@typescript-eslint/parser": "5.60.0",
     "eslint": "8.41.0",
     "eslint-config-prettier": "8.6.0",
-    "eslint-import-resolver-custom-alias": "1.3.0",
+    "eslint-import-resolver-typescript": "3.5.5",
     "eslint-plugin-eslint-comments": "3.2.0",
-    "eslint-plugin-jest": "27.2.1",
     "eslint-plugin-import": "2.27.5",
+    "eslint-plugin-jest": "27.2.1",
     "eslint-plugin-prettier": "4.2.1",
     "eslint-plugin-tsdoc": "0.2.17",
     "eslint-plugin-unicorn": "42.0.0",
     "eslint-plugin-vue": "9.9.0",
     "eslint-plugin-vuejs-accessibility": "1.1.1",
```

Comment:
Theoretically (potentially _ideally_) we could move our entire ESLint configuration into the plugin with our root `eslintrc` merely importing the plugin. That would allow us to get rid of all the ESLint dependencies (except ESLint and the openverse eslint plugin) in the root package.json and make things a bit cleaner here.

That's something for a future issue, however. Not worth adding the complexity to this PR.